plutus-benchmark/nofib/test/formulaBudget.budget.golden

```diff
@@ -1,3 +1,3 @@
-({ cpu: 37549556908
-| mem: 161728148
+({ cpu: 37605791908
```

Comment:
I didn't do a detailed investigation of these benchmarking numbers. But from the `plutus-tx-tests` test cases, it is clear that the PIRs are sometimes different due to the different GHC Cores, so there's nothing unexpected about this.